extra_data/stacking.py

```diff
@@ -131,6 +131,42 @@ def stack_detector_data(train, data, axis=-3, modules=16, fillvalue=None,
     return stack
 
 
+def stack_from_xarray(xrdata, axis=-3, modules=16, fillvalue=np.nan,
```

Comment:
Rather than taking an `axis=` parameter, I would suggest that we check which dimension is labelled `module` in the DataArray, and use its position, so the array (or `StackView`) has its axes in the same order as the input.